### ISSUE TITLE
[MUST] Use open source dependencies

### ISSUE THREAD

**Issue by jason-fox** [OPSILab/Idra#14]
Several dependencies have been identified which do not have a `LICENSE` and therefore need to be removed or updated to licensed components:

```
github.com/OPSILab/Idra  angular-ui-ace
github.com/OPSILab/Idra  angular-ui-layout
github.com/OPSILab/Idra  COS
```

Elements without a `LICENSE` are technically _All rights reserved_ and should not be used in Open Source

- MUST requirement from the TSC

**Comment by robcalla** [comment]
Hi Jason, thank you for the comment.
Could you please give me some insight into this issue? Since both [angular-ui-ace](https://github.com/angular-ui/ui-ace) and [angular-ui-layout](https://github.com/angular-ui/ui-layout) have a `LICENSE` and it is MIT.
Moreover, we will change the [COS](https://mvnrepository.com/artifact/servlets.com/cos) mvn dependency.

**Comment by jason-fox** [comment]
I've just been running the [FOSSA Bot](https://app.fossa.io/projects/git%2Bgithub.com%2FFiware%2Fcatalogue/refs/branch/master/024fd5041d146971eb34fa6a546f85bebf0af065/issues/licensing?filter=unlicensed_dependency&revisionScanId=393148) over the whole FIWARE Catalog - maybe its a false positive, maybe the version needs updating or maybe its a dependency of the dependencies.

**Comment by robcalla** [comment]
Ok, I've found the problem. It is related to the version of the libraries we are currently using. I'm going to change the versions and create a PR asap.

**Comment by robcalla** [comment]
I'm closing this issue since it has been solved with PR #15.
Please let us know if you find any other issue.
